src/System.Runtime.InteropServices.RuntimeInformation/ref/System.Runtime.InteropServices.RuntimeInformation.cs

```diff
@@ -21,8 +21,20 @@ public partial struct OSPlatform : System.IEquatable<System.Runtime.InteropServi
         public static bool operator !=(System.Runtime.InteropServices.OSPlatform left, System.Runtime.InteropServices.OSPlatform right) { return default(bool); }
         public override string ToString() { return default(string); }
     }
+
+    public enum Architecture
+    {
+        x86,
+        x64,
+        ARM
+    }
+
     public static partial class RuntimeInformation
     {
+        public static Architecture GetProcessArchitecture() { return default(Architecture); }
+        public static Architecture GetOSArchitecture() { return default(Architecture); }
+        public static string GetOSDebugName() { return default(string); }
```

Comment:
Apart from name, there was another suggestion to make these properties. Properties are better here, since these values don't change with subsequent calls, and they just represent some constant data about the environment.